constants.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 # SELENIUM CONSTANTS
-saveDir =   "{0}{1}data{1}".format(os.path.dirname(os.path.realpath(__file__)), os.sep)
+saveDir = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
```

Comment:
this needs to be wrapped in another os.path.join, that's why os.sep was used before (not sure about efficiency between format and os.path.join tbh, but using format just seemed cleaner), otherwise stats.py need to be changed as well